content/it/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Controller
+id: controller
+date: 2018-04-12
+full_link: /docs/concepts/architecture/controller/
+short_description: >
+  Un circuito di controllo che osserva lo stato condiviso del cluster attraverso l'API server and apporta le modifiche necessarie per portate lo stato corrente verso lo stato desiderato.
```

Comment:
WDYT about
```suggestion
  Un software che implementa un control loop che osserva lo stato condiviso del cluster attraverso l'API server and apporta le modifiche necessarie per portate lo stato corrente verso lo stato desiderato.
```

Reply:
partially changed due to previous translation agreement